### DISCUSSION TITLE
1.21 release

### DISCUSSION THREAD

**Discussion by ConnerGits** [maruohon/minihud#350] [Q&A]
Good day! I have a question about when, if at all, minihud for 1.21 would be released. I don't mean to rush it in any way, I understand that modding takes a lot of time and you're doing it for no guaranteed monetary gain. But I enjoy the mod a lot so I wanted to ask. Thank you!

**Comment by fliplus** [comment]
Masa is not updating the mod himself as stated [here](https://discord.com/channels/169369095538606080/779705877695430687/1248282348693229658). You can find 1.20.5+ ports in [Sakura-Ryoko’s GitHub repositories](https://github.com/sakura-ryoko?tab=repositories).

**Reply by ConnerGits** [reply]
Awesome, thank you! I didn't see the post before.
